l3kernel/l3keys.dtx

```diff
@@ -2559,6 +2630,28 @@
 % \end{macro}
 % \end{macro}
 %
+% \subsection{Key properties for \LaTeXe{} options}
+%
```

Comment:
could do with a few words on where they are supposed to be used, but perhaps that is too early if we haven't decided on the final name for `\SetKeys`

Reply:
Yes, I'm not sure if this even lives in `l3keys`: we could have some method for aliasing property names, which would then allow it to be handled in `ltkeys` (sill to be written).

Reply:
yes, should perhaps moved there (when written)